### ISSUE TITLE
Pre-Draft: Adding Nuo.Network

### ISSUE THREAD

**Issue by DunnCreativeSS** [Stakedllc/YIPs#20]
Why
Adding nuo.network would allow for more potential yields should their rates for any assets be higher.

Solution
I think nuo.network and has non-verified source code that prevents us from integrating on-chain. This is a blocker.

https://github.com/NuoNetwork

**Comment by DunnCreativeSS** [comment]
Refer here for existing conversation: https://github.com/Stakedllc/robo-advisor-yield/issues/5

**Comment by dpurhar27** [comment]
Correct, their smart contracts don't allow other smart contracts to interact with them, so it's impossible to integrate with them until they remove this barrier. They've mentioned an upgrade that'll do that with a backwards moving timeline since May 2019. 

Therefore, Nuo support is something to keep an eye on, but not actively plan around quite yet. There are also some other issues with Nuo - ie. their reported interest rates are approximations and not guaranteed. The latter is not a blocker, and solutions can be made to accommodate for it. The former is a blocker.
